minor testcases added (#1496)
codex-team/editor.js: test/testcases.md

    - [ ] If omitted the caret shouldn't be placed anywhere.

- [ ] `placeholder` property
  - [ ] With the empty editor
    - [ ] If `string` passed, the string should be placed as placeholder to the first empty block only.
    - [ ] If `false` passed, the first empty block should be placed without placeholder.
    - [ ] If omitted the first empty block should be placed without placeholder.

- [ ] `minHeight` property
    - [ ] If `number` passed, the height of editor's bottom area from last Block should be the `number`.
    - [ ] If omitted the height of editor's bottom area from last Block should be the default `300`.

- [ ] `logLevel` property
  - [ ] If `VERBOSE` passed,the editor should outputs all messages to the console.
  - [ ] If `INFO` passed,the editor should outputs info and debug messages to the console.
  - [ ] If `WARN` passed,the editor should outputs only warn messages to the console.
  - [ ] If `ERROR` passed,the editor should outputs only error messages to the console.
  - [ ] If omitted the editor should outputs all messages to the console.

- [ ] `defaultBlock` property
  - [ ] If `string` passed

>     - [ ] If passed `string` in tools,the passed tool should be used as default tool.

Reviewer: ```suggestion
    - [ ] If passed `string` in the `tools` option, the passed tool should be used as the default tool.
```

>     - [ ] If passed `string` not in tools,the Paragraph tool should be used as default tool.

Reviewer: ```suggestion
    - [ ] If passed `string` not in the `tools` option, the Paragraph tool should be used as the default tool.
```

  - [ ] If omitted the Paragraph tool should be used as default tool.

- [ ] `sanitizer` property
  - [ ] If `object` passed

>     -[ ] The Editor.js should be clean the HTML tag according to mentioned configuration.

Reviewer: ```suggestion
    - [ ] The Editor.js should clean the HTML tags according to mentioned configuration.
```

>   - [ ] If omitted the Editor.js should be initialize with default `sanitizer` configuration, which allow the tags like `paragraph`, `anchor` and `bold` for cleanning HTML.

Reviewer: ```suggestion
  - [ ] If omitted the Editor.js should be initialized with the default `sanitizer` configuration, which allows the tags like `paragraph`, `anchor`, and `bold` for cleaning HTML.
```

  
- [ ] `tools` property

>   - [ ] If `object` passed

Reviewer: I think, we also need to check the structure of the `tools` available options

Author: @neSpecc can you describe it with some examaple.

>     -[ ] Editor.js should be initialized with the Passed tools only.

Reviewer: ```suggestion
    - [ ] Editor.js should be initialized with the Passed tools only.
```

>   - [ ] If omitted Editor.js should be initialized with the Paragraph tool only.

Reviewer: ```suggestion
  - [ ] If omitted 
    - [ ] Editor.js should be initialized with the Paragraph tool only.
```
